lib/pbench/cli/server/shell.py

```diff
@@ -40,7 +39,7 @@ def main():
             "--pid",
             "/run/pbench-server/gunicorn.pid",
             "--bind",
-            f"{host}:{port}",
+            f"localhost:${port}",
```

Comment:
Does `localhost` works? The bind host used to be 0.0.0.0

Reply:
That's the point; gunicorn is only listening on localhost now, relying on Apache to act as gateway from the external network. From the outside, you only see port 80. (Or, when we get to https, 433.) Pbench and gunicorn don't care; it's all localhost to us. Kinda like being inside a container; which, eventually, we probably will be.

Reply:
Well, this requires us to run Apache locally to reverse proxy it, but we run nginx remotely to reverse proxy services.  So ideally we'd like to be able to specify the bind IP and port # together that should be used by the server.

Reply:
Yes; the root point here was that the main html document loaded for the dashboard can query its own origin on port 80 to hit the configuration API.

But, yes, I hadn't thought about your current remote NGINX proxy configuration, which can be configured to do the same thing... but would definitely require opening the API port.

OK, so I'll revert this change; to work in the local Apache proxy mode, you just set `pbench-server.cfg` to listen on `localhost` instead of hardcoding it here. For a remote proxy, you can still specify the hostname and port and manually open the firewall.

That'll work, too.

I suppose a bigger question is whether it's OK to create the Apache proxy config file in the server httpd activate Ansible role. Under the current production/staging server deployment we're running Apache locally and serving the dashboard even though we're using the remote NGINX proxy; would someone reasonably expect to use the standard Ansible config script here? Maybe... in which case I need to either split it into a separate playbook, or make it conditional on something in the inventory file.